test/test_nn.py

```diff
@@ -14500,6 +14500,24 @@ def test(nonlinearity, *args, **kwargs):
         test('threshold', 3, 2)
         test('threshold', 3, 2, inplace=True)
 
+    def test_rrelu_bounds(self, device):
+        def test(inplace, module, input, lower, upper):
+            with self.assertRaisesRegex(RuntimeError, "Lower bound must be less than the upper bound"):
+                if inplace is None:
+                    module(input, lower, upper)
+                elif input is None:
+                    module(lower, upper, inplace)(torch.rand((2)))
+                module(input, lower, upper, inplace)
+
+        input = torch.rand((3, 3), device=device, dtype=torch.float32)
+        lower, upper = 0.4, 0.1
+
+        test(False, nn.RReLU, None, lower, upper)
+        test(True, nn.RReLU, None, lower, upper)
+        test(False, nn.functional.rrelu, input, lower, upper)
```

Comment:
Can this test just be an ErrorInput for `nn.functional.rrelu`? The module just calls the function:

https://github.com/pytorch/pytorch/blob/ebd4e90ff7f6d1e57bbf6f2e717a8addb6b28e28/torch/nn/modules/activation.py#L167

Reply:
Sure, removed these tests in favor of ErrorInputs.